{"Desktop\\Work\\tensorflow-master\\tensorflow\\python\\eager\\execute": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\eager\\execute.quick_execute": ["tensorflow.python.eager.core._SymbolicException", "tensorflow.python.framework.ops._is_keras_symbolic_tensor", "tensorflow.python.pywrap_tfe.TFE_Py_Execute", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.pywrap_tfe.TFE_Py_Execute": [], "tensorflow.python.eager.core._status_to_exception": [], "tensorflow.python.framework.ops._is_keras_symbolic_tensor": [], "tensorflow.python.eager.core._SymbolicException": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\eager\\execute.execute_with_cancellation": ["tensorflow.python.eager.core._SymbolicException", "tensorflow.python.framework.ops._is_keras_symbolic_tensor", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.pywrap_tfe.TFE_Py_ExecuteCancelable"], "tensorflow.python.pywrap_tfe.TFE_Py_ExecuteCancelable": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\eager\\execute.execute_with_callbacks": ["<builtin>.tuple", "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\eager\\execute.quick_execute"], "<builtin>.tuple": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\eager\\execute.must_record_gradient": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\eager\\execute.record_gradient": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\eager\\execute.make_float": ["<builtin>.float", "<builtin>.repr", "<builtin>.isinstance", "<builtin>.TypeError"], "<builtin>.isinstance": [], "<builtin>.repr": [], "<builtin>.TypeError": [], "<builtin>.float": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\eager\\execute.make_int": ["<builtin>.repr", "<builtin>.int", "<builtin>.isinstance", "<builtin>.TypeError"], "<builtin>.int": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\eager\\execute.make_str": ["<builtin>.repr", "tensorflow.python.util.compat.as_bytes", "<builtin>.isinstance", "<builtin>.TypeError"], "tensorflow.python.util.compat.as_bytes": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\eager\\execute.make_bool": ["<builtin>.repr", "<builtin>.isinstance", "<builtin>.TypeError"], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\eager\\execute.make_type": ["tensorflow.python.framework.dtypes.as_dtype", "<builtin>.repr", "<builtin>.TypeError"], "tensorflow.python.framework.dtypes.as_dtype": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\eager\\execute.make_shape": ["<builtin>.ValueError", "tensorflow.python.framework.tensor_shape.as_shape", "<builtin>.TypeError"], "tensorflow.python.framework.tensor_shape.as_shape": [], "<builtin>.ValueError": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\eager\\execute.make_tensor": ["<builtin>.isinstance", "<builtin>.TypeError", "google.protobuf.text_format.Merge", "tensorflow.core.framework.tensor_pb2.TensorProto", "<builtin>.repr"], "tensorflow.core.framework.tensor_pb2.TensorProto": [], "google.protobuf.text_format.Merge": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\eager\\execute.args_to_matching_eager": ["tensorflow.python.eager.core._SymbolicException", "<builtin>.isinstance", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.framework.ops._is_keras_symbolic_tensor"], "tensorflow.python.framework.ops.convert_to_tensor": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\eager\\execute.convert_to_mixed_eager_tensors": ["tensorflow.python.framework.ops.convert_to_tensor"], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\eager\\execute.args_to_mixed_eager_tensors": ["<builtin>.len", "<builtin>.isinstance", "<builtin>.range", "<builtin>.ValueError", "tensorflow.python.framework.ops.convert_to_tensor"], "<builtin>.len": [], "<builtin>.range": []}
